Client/Frontend/Browser/TabManager.swift

```diff
@@ -836,15 +841,15 @@ class TabManager: NSObject {
         }
     }
 
-    lazy var restoreAllTabs: () -> Tab = {
+    func restoreAllTabs() -> Tab {
```

Comment:
Needs _run once_ protection too.

Reply:
This does not require a run once logic as the underlying logic is already run once, which on other case will return nil and here it will add a new tab. If we to a run once here then too we have to create a new tab.